apps/console/src/components/pages/protected/standards/standard-details-accordion.tsx

```diff
@@ -178,7 +178,7 @@ const StandardDetailsAccordion: FC = () => {
                     setIsDialogOpen(true)
                   }}
                 >
-                  Add to Organization ({selectedInCategory.length})
+                  Add Controls ({selectedInCategory.length})
```

Comment:
I think this text should stay the same, just the dialog header updated like you have above. 
```suggestion
                  Add to Organization ({selectedInCategory.length})
```